src/Sylius/Behat/Context/Ui/Shop/LoginContext.php

```diff
@@ -285,6 +285,22 @@ public function iShouldNotBeAbleToChangeMyPasswordAgainWithTheSameToken(): void
     {
         $this->resetPasswordPage->tryToOpen(['token' => 'itotallyforgotmypassword']);
 
+        $this->iShouldNotBeAbleToChangeMyPassword();
+    }
+
+    /**
+     * @Then I should not be able to change my password with this token
+     */
+    public function iShouldNotBeAbleToChangeMyPasswordWithThisToken(): void
+    {
+        $this->iShouldNotBeAbleToChangeMyPassword();
+    }
+
+    /**
+     * @Then I should not be able to change my password
+     */
+    public function iShouldNotBeAbleToChangeMyPassword(): void
+    {
```

Comment:
```suggestion
    /**
     * @Then I should not be able to change my password
     * @Then I should not be able to change my password with this token
     */
    public function iShouldNotBeAbleToChangeMyPassword(): void
    {
```

As it does the same thing, we can just add another `@Then` annotation to the existing method.